install-and-test.sh

```diff
@@ -1,5 +1,6 @@
+#!/bin/sh
+
 dir=$(pwd)
-CI=true
```

Comment:
It looks like this variable assignment has no effect (from my quick testing), so I removed it.